doc/man/8/rados.rst

```diff
@@ -26,6 +26,12 @@ Options
 
    Interact with the given pool. Required by most commands.
 
+.. option:: --pgid
+
+   As an alternative to "--pool", "--pgid" also allows user to specify the
+   PG id to which the command will be directed. With this option, certain
+   commands like "ls" allows user to limit the scope of the command to the given PG.
+
```

Comment:
```diff
--- a/doc/man/8/rados.rst
+++ b/doc/man/8/rados.rst
@@ -28,9 +28,9 @@ Options

 .. option:: --pgid

-   As an alternative to "--pool", "--pgid" also allows user to specify the
+   As an alternative to ``--pool``, ``--pgid`` also allows user to specify the
    PG id to which the command will be directed. With this option, certain
-   commands like "ls" allows user to limit the scope of the command to the given PG.
+   commands like ``ls`` allows user to limit the scope of the command to the given PG.

 .. option:: -s snap, --snap snap
```

Reply:
sorry Kefu, misunderstood the quotes, fixed it. I have also verified with hint given in the comment - make manpages and it looks good.